common/changes/@microsoft/api-extractor/fwienber-api-extractor-fix-3593-reference-alias_2022-08-29-07-53.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Fix an issue where aliased classes sometimes had incorrect canonical references in *.api.json (GitHub  #3593)",
+      "type": "minor"
```

Comment:
Although this is technically a bug fix, I've converted it to a "minor" version bump because **DeclarationReferenceGenerator.ts** has lots of edge cases that are difficult to test comprehensively.